app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningDatasetSelectionFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.github.HumanLearning2021.HumanLearningApp.view.learning
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.activity.OnBackPressedCallback
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.fragment.app.findFragment
+import androidx.navigation.fragment.findNavController
+import com.github.HumanLearning2021.HumanLearningApp.R
+import com.github.HumanLearning2021.HumanLearningApp.databinding.FragmentLearningDatasetSelectionBinding
+import com.github.HumanLearning2021.HumanLearningApp.view.dataset_list_fragment.DatasetListWidget
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class LearningDatasetSelectionFragment : Fragment() {
+    private lateinit var parentActivity: FragmentActivity
+
+    private var _binding: FragmentLearningDatasetSelectionBinding?= null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        parentActivity = requireActivity()
+        _binding = FragmentLearningDatasetSelectionBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+
+        //TODO: use binding
```

Comment:
still relevant?

Reply:
are you referring to the binding comment?

Reply:
yup

Reply:
okay then I deleted it.